shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -2,21 +2,26 @@ import * as React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
 import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
 import * as Types from '../../constants/types/teams'
+import * as ChatTypes from '../../constants/types/chat2'
 import * as TeamsGen from '../../actions/teams-gen'
 import * as RPCGen from '../../constants/types/rpc-gen'
+import * as RPCChatGen from '../../constants/types/rpc-chat-gen'
 import {appendNewTeamBuilder, appendTeamsContactsTeamBuilder} from '../../actions/typed-routes'
 import capitalize from 'lodash/capitalize'
 import {FloatingRolePicker} from '../role-picker'
-import {ModalTitle} from '../common'
+import {useDefaultChannels} from '../team/settings-tab/default-channels'
+import {ModalTitle, ChannelsWidget} from '../common'
 import {pluralize} from '../../util/string'
 import logger from '../../logger'
 
 const AddMembersConfirm = () => {
   const dispatch = Container.useDispatch()
 
-  const {teamID, addingMembers} = Container.useSelector(s => s.teams.addMembersWizard)
+  const {teamID, addingMembers, defaultChannels} = Container.useSelector(s => s.teams.addMembersWizard)
   const fromNewTeamWizard = teamID === Types.newTeamWizardTeamID
+  const isBigTeam = Container.useSelector(s => (fromNewTeamWizard ? false : Constants.isBigTeam(s, teamID)))
```

Comment:
why not allow adding to channels if coming from the new team wizard? We could make `Constants.isBigTeam` aware of `Types.newTeamWizardTeamID` if that seems cleaner.

Reply:
Channels in the new team wizard don't exist yet, so I think the changes would be more far reaching. `useAllChannelMetas` would need to know to pull from the store instead. Or `ChannelsWidget` could be made to know about it and handle it. Then here we wouldn't call `bulkAddToManyConvs` in the confirm screen (TRIAGE-2531 is related). I'd prefer to have logic related to new teams not leak out of the wizard too much. Anyway I'm not saying we shouldn't do it, but it seems big enough to be considered alone.

Reply:
Seems reasonable. Maybe we can talk about with design and then make another ticket to do that in a clean way if it seems warranted

Reply:
Made Y2K-1699 since it seems like a nice feature and uses existing design.